var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -42,7 +42,7 @@ class Nwchem(Package):
     depends_on("mpi")
     depends_on("scalapack")
     depends_on("fftw-api")
```

Comment:
When trying to build it without fftw installed in the host OS, I get missing lib for `-lfftw3`, so I propose:
```suggestion
    depends_on("fftw")
    depends_on("fftw-api")
```

Reply:
I didn't run into this yesterday, and I'm building in a very sparse container. What've you got setup for providers for fftw-api?

Reply:
@snehring: Somehow `depends_on("fftw-api")` didn't add `fftw` into the spec in that install.

It's unrelated to this PR and a completely different issue.